### ISSUE TITLE
Fine-tuning MLM Module on Homologous Sequence

### ISSUE THREAD

**Issue by OrangeCat7777777** [ai4protein/Pro-Prime#3]
I'm curious about how the model is finetuned to  be applied on specific protein design problems. From the "Fine-tuning MLM Module on Homologous Sequence" section of your paper, it seems that the model is fine-tuned with MLM loss:

1. Is the other two losses, the OGT prediction loss and the correlation loss, involved in this fine-tuning process?

2. How to decide when to stop fine-tuning? Do you split a validation set?

Thanks in advance!

**Comment by ai4protein** [comment]
1. 同源序列微调的时候仅用了MLM LOSS
2. 根据你的蛋白不同，一般来说1-2个epoch足够

这些非代码的问题，请给我们发送邮件解决，如果需要同源序列微调的代码，请写信说明单位，我们可以提供一份拷贝。
